tests/async.rs

```diff
@@ -93,24 +93,27 @@ where
 
     fn test<C, F>(self, c: C)
     where
-        F: IntoFuture<Item = (), Error = mysql_async::errors::Error>,
+        F: IntoFuture<Item = (), Error = mysql_async::error::Error>,
         C: FnOnce(mysql_async::Conn) -> F,
     {
+        let mut runtime = current_thread::Runtime::new().unwrap();
```

Comment:
Let's construct this down where the `Core` used to be constructed just to keep the diff as small as possible. I don't think we need the runtime all the way up here?